addons.php

```diff
@@ -0,0 +1,76 @@
+<?php
+//official ldap_escape function
+if (!function_exists('ldap_escape')) {
```

Comment:
This should just print a message saying that rendezvous does not support php versions < 5.6.0.

Reply:
Actually ldap_escape function is not available in php versions > 5.6.0 as standalone because it's not officially added in linux's php package. It requires another package called php-ldap (`sudo apt-get install php-ldap`) in order to work properly. Unless I'm mistaken, then I could make another commit.

Reply:
Maybe we should add this package as a prerequisite for rendezvous then.

Reply:
Will do.